frontend/src/data/variables/AcsPopulationProvider.ts

```diff
@@ -61,6 +61,30 @@ class AcsPopulationProvider extends VariableProvider {
         population_pct: (row) => percent(row.population, totalStatePopulation),
       });
     });
+
+    // Race data comes from backend with Total, if it's not wanted, we remove it.
+    if (
+      !breakdowns.includeTotal &&
+      (breakdowns.demographic === "race_nonstandard" ||
+        breakdowns.demographic === "race")
+    ) {
+      df = df.where((row) => row["race_and_ethnicity"] !== "Total");
+    }
+
+    // Age and sex data comes from backend without Total, if it's wanted, we add it.
+    if (
+      breakdowns.includeTotal &&
+      (breakdowns.demographic === "age" || breakdowns.demographic === "sex")
+    ) {
+      df = df.concat(
```

Comment:
resetIndex after concat to be safe since the indices overlap when you concat frames